lib/razor/command/create_broker.rb

```diff
@@ -64,7 +63,8 @@ def run(request, data)
   def self.conform!(data)
     data.tap do |_|
       # Allow "c" as a shorthand.
-      add_hash_alias(data, 'configuration', 'c')
+      add_hash_alias(data, 'c', 'configuration')
+      add_alias(data, 'broker-type', 'broker_type')
```

Comment:
Instead of requiring that command authors remember to set up aliases for the attributes that have '_' in them, is there a way we could make that automatic ? IOW, if you say "attr 'foo_bar'" that the machinery automatically sets up an alias for 'foo-bar' ?